src/routes/sign-up/steps/modal-stub.jsx

```diff
@@ -0,0 +1,25 @@
+//assets/form/SeedModalForm.jsx
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+import Row from "react-bootstrap/Row";
+
+//NOTE: This is a temporary component to be replaced with the modal component
```

Comment:
Modal has been merged, so I believe you can go ahead and do this replacement

Reply:
I sent you a message about this - was only working on the Layout Component, was going to work on the Wizard component after the layout component is merged and correct (less revisions and quicker time for Layout to be available).  Should I delete the work done on the wizard for this PR?

Reply:
removed sign-up folder (it is in the feature/wizard-component branch).  Will work on wizard-component independently from this branch